### PR TITLE
Added initial structure for deploying sign and verify service

### PR DESCRIPTION
**Pull request by blarghmatey**
The sign and verify service will be deployed on AWS Fargate along with a proxy that will validate an HMAC signature for
POST requests. This adds the resources for:
- defining a secrets manager secret containing the unlocked DID document
- creating a Fargate cluster
- defining the task for the fargate cluster
- defining a service using the fargate task
- generating the role policy to allow access to the secrets manager secret
- creates and configures a load balancer and ACM certificate
- registers the service with DNS

### CODE REVIEW

File: src/ol_infrastructure/applications/sign_and_verify/__main__.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                    {'name': 'UNLOCKED_DID', 'valueFrom': arn}
+                ],
+                'portMappings': [
+                    {'containerPort': CONTAINER_PORT, 'protocol': 'tcp'}
+                ],
+                'logConfiguration': {
+                    'logDriver': 'awslogs',
+                    'options': {
+                        'awslogs-group': f'digital-credentials-sign-and-verify-{env_suffix}',
+                        'awslogs-region': 'us-east-q',
```
Comment (rhysyngsun) [inline src/ol_infrastructure/applications/sign_and_verify/__main__.py:129]: Typo?
```suggestion
                        'awslogs-region': 'us-east-1',
```

**Review by rhysyngsun — COMMENTED** [review]
Generally looks good

File: src/ol_infrastructure/applications/sign_and_verify/__main__.py [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    description='Base64 encoded JSON object of the Unlocked DID that specifies the signing keys '
+    'for the digital credentials sign and verify service.',
+    name_prefix=f'sign-and-verify-unlocked-did-{env_suffix}',
+    tags=aws_config.tags
+)
+
+unlocked_did_secret_value = secretsmanager.SecretVersion(
+    f'sign-and-verify-unlocked-did-value-{env_suffix}',
+    secret_id=unlocked_did_secret.id,
+    secret_string=sign_and_verify_config.require_secret('unlocked_did'),  # Base64 encoded JSON object of unlocked DID
```
Comment (shaidar) [inline src/ol_infrastructure/applications/sign_and_verify/__main__.py:96]: Should this be `unlocked_did_secret`?

Comment (blarghmatey) [reply src/ol_infrastructure/applications/sign_and_verify/__main__.py:97]: No, this is referring to the config entry defined here https://github.com/mitodl/ol-infrastructure/pull/19/files#diff-e32743a26673c7e2751688227fe2791404d7dad3865c754decb76be296c97af5R9

**Comment by shaidar** [comment]
👍

**Pull request merged**